docs/overview.rst

```diff
@@ -181,23 +176,32 @@ or HPy Hybrid ABIs require installing the ``hpy.universal`` package on
 CPython and have no further requirements on Pythons that support HPy natively.
 
 
-C extensions
-------------
+Benefits for the Python ecosystem
+---------------------------------
 
-If you are writing a Python extension in C, you are a consumer of the HPy
-API. There are three big advantages in using HPy instead of the old Python/C
-API:
-
-  - Speed on PyPy, GraalPython and other alternative implementations: according
-    to early :ref:`benchmarks`, an extension written in HPy can be ~3x faster
-    than the equivalent extension written in Python/C.
+The HPy project offers some benefits to the python ecosystem, both to Python
+users and to library developers.
 
+  - C extensions can achieve much better speed on alternative implementions,
+    including PyPy and GraalPython: according to early :ref:`benchmarks`, an
+    extension written in HPy can be ~3x faster than the equivalent extension
+    written in Python/C.
   - Improved debugging: when you load extensions in :ref:`debug-mode:debug mode`,
     many common mistakes are checked and reported automatically.
-
-  - Universal binaries: you can choose to distribute only Universal ABI
-    binaries. This comes with a small speed penalty on CPython, but for
-    non-performance critical libraries it might still be a good tradeoff.
+  - Universal binaries: libraries can choose to distribute only Universal ABI
+    binaries. By doing so, they can support all versions of CPython for which
+    an hpy loader exists, including versions of CPython that have not yet been
+    developed. This currently comes with a small speed penalty on CPython, but
+    for non-performance critical libraries it might still be a good tradeoff.
```

Comment:
The universal binaries also work on any other Python, like PyPy or GraalPython. Even the original text is advertising that clearly, but maybe a chance to improve that.